tt_metal/hw/firmware/src/erisc.cc

```diff
@@ -114,6 +116,7 @@ void __attribute__((section("erisc_l1_code"))) ApplicationHandler(void) {
         if (erisc_info->launch_user_kernel == 1) {
             kernel_profiler::init_profiler();
             kernel_profiler::mark_time(CC_MAIN_START);
+            DEBUG_STATUS('K');
```

Comment:
take a look in, eg, brisc.cc and make these statuses match those, eg, 'R' for run and 'D' for done

Reply:
Updated these messages in latest commit